Update redirects.asciidoc (#84719)
elastic/kibana: docs/redirects.asciidoc

=== Variables

This page has moved. Refer to <<url-template-variables>>.

[float]
[[time-series-visual-builder]]
=== Time Series Visual Builder

This page was deleted. Please see <<Dashboard>>.

[float]

> [[development-functional-tests]]

Reviewer: This is in the docs - https://www.elastic.co/guide/en/kibana/current/development-tests.html#development-functional-tests. You can remove lines 203-207.

=== Development Functional Tests

This page has been moved. Please see <<Testing>>.

[float]
[[kibana-keystore-has-moved-from-the-data-folder-to-the-config-folder]]
=== Kibana Keystore has moved from the Data Folder to the Config Folder

> This page has been deleted. Please see <<Breaking Changes in 8.0>>.

Reviewer: ```suggestion
This page has been deleted. Please see link:https://www.elastic.co/guide/en/kibana/7.9/breaking-changes-7.9.html#user-facing-changes-79[Breaking changes in 7.9].
```

[float]
[[createvis]]
=== Create Visualization

> This page has been deleted. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been deleted. Please see <<dashboard>>.
```

[float]
[[data-table]]
=== Data Table

> This page has been deleted. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been deleted. Please see <<dashboard>>.
```

[float]
[[xy-chart]]
=== Line, Area, and Bar Chart

> This page has been deleted. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been deleted. Please see <<dashboard>>.
```

[float]
[[watcher-create-threshold-alert]]
=== Create Threshold Alert

> This page has been moved. Please see <<Watcher>>.

Reviewer: This content should be combined with the existing Watcher content on line 84.

[float]
[[add-canvas-events]]
=== Add Canvas Elements

> This page has been moved. Please see <<Canvas>>.

Reviewer: ```suggestion
This page has been moved. Please see <<canvas>>.
```

[float]
[[vega-lite-tutorial]]
=== Vega-Lite Tutorial

> This page has been moved. Please see <<Create your first visualization with Vega-Lite>>.

Reviewer: ```suggestion
This page has been moved. Please see <<vega-lite-tutorial-create-your-first-visualizations>>.
```

[float]
[[heatmap-chart]]
=== Heatmap Chart

> This page has been moved. Please see <<Heat map layer>>.

Reviewer: ```suggestion
This page has been moved. Please see <<dashboard>>.
```

[float]
[[interface-overview]]
=== Interface Overview

> This page has been moved. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been moved. Please see <<dashboard>>.
```

[float]
[[time-series-visualizations]]
=== Featured Visualizations

> This page has been moved. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been moved. Please see <<dashboard>>.
```

[float]
[[timelion-customize]]
=== Customize and format visualizations

> This page has been moved. Please see <<Dashboard>>.

Reviewer: ```suggestion
This page has been moved. Please see <<dashboard>>.
```

[float]
[[dashboard-drilldown]]
=== Dashboard Drilldowns

> This page has been moved. Please see <<Create custom dashboard actions>>.

Reviewer: ```suggestion
This page has been moved. Please see <<dashboard-drilldowns>>.
```

> [float]

Reviewer: This content exists in the Kibana docs - https://www.elastic.co/guide/en/kibana/master/rpm.html. You can remove lines 282-286.

[[rpm]]
=== Install Kibana with RPM

This page has been moved. Please see <<Install Kibana with RPM>>.

> [float]

Reviewer: This content exists in the Kibana docs - https://www.elastic.co/guide/en/kibana/master/automating-report-generation.html. You can remove lines 288-292.

[[automating-report-generation]]
=== Automating Report Generation

This page has been moved. Please see <<Automating Report generation>>.

[float]
[[development-plugin-localization]]
=== Localization for plugins

> This page has been moved. Please see <<Localization for plugins outside the Kibana repo>>.

Reviewer: ```suggestion
This page has been moved. Please see <<external-plugin-localization>>.
```

This page has moved. Refer to <<url-template-variables>>.

[float]
[[time-series-visual-builder]]
=== Time Series Visual Builder

This page was deleted. Refer to <<dashboard>>.

> [float]

Reviewer: This link still exists in the docs. You can delete lines 203-207.

[[development-functional-tests]]
=== Development Functional Tests

This page has been moved. Refer to <<Testing>>.

[float]
[[kibana-keystore-has-moved-from-the-data-folder-to-the-config-folder]]
=== Kibana Keystore has moved from the Data Folder to the Config Folder

This page has been deleted. Refer to link:https://www.elastic.co/guide/en/kibana/7.9/breaking-changes-7.9.html#user-facing-changes-79[Breaking changes in 7.9].

[float]
[[createvis]]
=== Create Visualization

This page has been deleted. Refer to <<dashboard>>.

[float]
[[data-table]]
=== Data Table

This page has been deleted. Refer to <<dashboard>>.

[float]
[[xy-chart]]
=== Line, Area, and Bar Chart

This page has been deleted. Refer to <<dashboard>>.

> [float]

Reviewer: This link exists in the docs. You can remove lines 234-238.
